apm-agent-builds/pom.xml

```diff
@@ -79,6 +79,11 @@
             <artifactId>apm-dubbo-plugin</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>apm-es-restclient-plugin-common</artifactId>
+            <version>${project.version}</version>
+        </dependency>
```

Comment:
[question] is this extra dependency a leftover ? From the changes in PR this does not seem to be needed.